File: AssayDashboard/Source/Hologic.AssayDashboard.AssayParser/Models/ResultCategory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Xml.Serialization;

namespace Hologic.AssayDashboard.AssayParser
{
    public class ResultCategory
    {
        [XmlElement("ResultCategoryName")]
        public string categoryName;

        [XmlElement("ResultCategoryIndex")]
        public int index;

        [XmlElement("IsSentToLIS")]
        public string sentToLIS;

        [XmlElement("IsSentToUI")]
        public string sentToUI;

        [XmlElement("IsSentToReport")]
        public string sentToReport;

        [XmlElement("IsLISHoldReleaseInterpretation")]
        public string lisHold;

        [XmlElement("IsIncludedInSpecimenTallies")]
        public string inSpecimenTallies;

        [XmlElement("ResultCategoryEnumeration")]
        public ResultEnum resultEnum;
        public bool active;

        public ResultCategory()
        {
            this.categoryName = "";
            this.index = 0;
            this.sentToLIS = "";
            this.sentToUI = "";
            this.sentToReport = "";
            this.lisHold = "";
            this.inSpecimenTallies = "";
            this.active = false; 
        }
    }
}

File: AssayDashboard/Source/Hologic.AssayDashboard.Client/src/app/assayviewrepo/assayviewrepo-models/resultcategory.ts
/*
// ----------------------------------------------------------------
// Copyright (c) 2017 Hologic, Inc.  All Rights Reserved.
// ----------------------------------------------------------------
*/
import { ResultEnumeration } from "app/assayviewrepo/assayviewrepo-models/resultenumeration";

export class ResultCategory {
    categoryName:string;
    index:string;
    sentToLIS: string;
    sentToUI: string;
    sentToReport:string;
    lisHold: string;
    inSpecimenTallies:string;
    resultEnum: ResultEnumeration;
    active: boolean;

    constructor() {
        this.categoryName = undefined;
        this.index = undefined;
        this.sentToLIS = undefined;
        this.sentToUI = undefined;
        this.lisHold = undefined;
        this.sentToReport = undefined;
        this.inSpecimenTallies = undefined;
        this.active = false;
    }
}